### ISSUE TITLE
Update custom email template with info about study and lab contact

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1359]
## TL;DR

We'd like to add some automated text to the 'custom email' template (the one used for researchers sending emails to participants) indicating that the message was sent on behalf of a researcher/lab about a specific study, and letting the participant know how to reply to the researcher/lab.

## Narrative

We recently had to change the way that we send 'custom emails', i.e. the messages sent from a study/researcher to participant. Specifically, these emails used to be sent from the lab email address but are now sent from the site's default MIT email, with the lab email as the reply-to address (see #1356). 

We don't want families to be confused about this change, so it might help to have some automated text at the start of each of these emails explaining that the message was sent on behalf of a specific study/researcher, and letting the family know how they can get in touch with the researcher/lab. There are a few added benefits of these changes:
- Sending this message from the site's email address makes it clearer to families that researchers sent the message _via the website_ (as opposed to getting ahold of their participant's email addresses and contacting them off the site, which is against our T&Cs).
- This will help families easily see what study the message is related to, since currently that information is not found in these emails unless the researcher remembers to add it to their custom message.

## List of changes

This is taken from @ianchandlercampbell's google doc. I will update this if that spec changes, and Ian please feel free to edit this or comment below if anything is incorrect.

1. In the custom email template, add the following:
    > This is an email for [child.name] from [lab.name] for [study.name]. You may reply to this email to message directly with the researchers at [lab.name] or email them at [lab.email] if you have other questions.
  
    (We'll try adding this both above and below the `{{ custom_message }}` and then decided which is better)
  
2. In the base email template (which is used for both custom emails and study announcement emails):
    - Add "CHS" to the update preferences text: 
      > "Update your CHS email preferences"

    - Under the unsubscribe link, add the linked text: 
      > "Questions or feedback for Children Helping Science?"
   
      which opens an email sent to childrenhelpingscience@gmail.com with the subject line "CHS Family Feedback or Question"

**Comment by becky-gilbert** [comment]
Update: this issue is on hold until we figure out whether/how to (1) handle emails sent to multiple families and (2) add the child's name to the view/message context.
